packages/react/src/FormErrorList/FormErrorList.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes } from 'react'
+import { Alert } from '../Alert'
+import { HeadingLevel } from '../Heading/Heading'
+import { LinkList } from '../LinkList'
+
+export type FormErrorListProps = {
+  errors: { id: string; label: string }[]
+  heading?: string
+  headingLevel?: HeadingLevel
+} & HTMLAttributes<HTMLDivElement>
+
+export const FormErrorList = forwardRef(
+  (
+    {
+      className,
+      errors,
+      heading = 'Verbeter de fouten voor u verder gaat',
```

Comment:
The heading is optional but you're giving it a default (dutch) value.

Reply:
I don't understand the comment? The component always has a heading, but you can optionally overwrite it.

Reply:
I mean the heading is optional for the alert but you're forcing it here.